open-company/open-company-web
# Attachments mobile fixes

Use a paperclip icon and a number only on mobile to show attachments.

To test:
- check FoC w/o attachments on desktop
- check FoC w/ attachments on desktop
- check FoC w/o attachments on mobile
- check FoC w/ attachments on mobile
- check expanded post w/o attachments on desktop (regression)
- check expanded post w/ attachments on desktop (regression)
- check expanded post w/o attachments on mobile (regression)
- check expanded post w/ attachments on mobile (regression)
